packages/apps/hr/src/features/expenseLineSlice.js

```diff
@@ -53,6 +103,32 @@ const expenseLineSlice = createSlice({
       isListEnd: 'isListEnd',
       list: 'expenseLineList',
     });
+    generateInifiniteScrollCases(builder, searchExpenseLineByIds, {
+      loading: 'loadingExpenseLineByIds',
+      moreLoading: 'moreLoadingExpenseLineByIds',
+      isListEnd: 'isListEndExpenseLineByIds',
+      list: 'expenseLineListByIds',
+    });
+    generateInifiniteScrollCases(
+      builder,
+      searchKilometricByIds,
+      {
+        total: 'totalNumberExpenseKilomectric',
+      },
+      {
+        manageTotal: true,
+      },
+    );
+    generateInifiniteScrollCases(
+      builder,
+      searchGeneralByIds,
+      {
+        total: 'totalNumberExpenseGeneral',
+      },
+      {
+        manageTotal: true,
+      },
+    );
```

Comment:
As said on another comment, those reducer are complicated for a poor utility for the moment. So for this case, please add management of list/loading props also.

For next time, when a reducer is only useful to get the number of items, please create it this way : 

```
    builder.addCase(function.pending, (state, action) => {
      state.loading= true;
    });
    builder.addCase(function.fulfilled, (state, action) => {
      state.loading= false;
      state.total = action.payload?.total;
    });
```
 It easier to understand what the reducer is used for.